_includes/tutorials/joining-stream-stream/flinksql/markup/dev/transient-join.adoc

```diff
@@ -0,0 +1,26 @@
+Let's join these event streams together to gain some insight into the order-to-shipping process.  Then we'll discuss some of the concepts used in the query to enable retrieving your desired results.
+NOTE: There are several columns in the results so it's best to set the result mode to `TABLEAU` using the following command:
+++++
+<pre class="snippet"><code class="sql"> SET sql-client.execution.result-mode=tableau; </code></pre>
```

Comment:
it's unfortunate that hljs doesn't do the right thing here and treats `sql` and `mode` as keywords and makes them bold:

![image](https://github.com/confluentinc/kafka-tutorials/assets/4550245/23e0fde9-1e25-4566-a51c-0abdf8d0fbf8)

Reply:
maybe use `plaintext` here instead. .. though I'm wondering how widespread an issue this is. could be a separate task to sweep the rest of KT